fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/postaggregation/FieldAccessorPostAggregation.java

```diff
@@ -24,14 +24,14 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(FieldAccessorPostAggregation.class);
 
-    private final Aggregation aggregation;
+    private final MetricField metricField;
 
     /**
      * Constructor.
      *
      * @param aggregation  Aggregation to access
      */
-    public FieldAccessorPostAggregation(@NotNull Aggregation aggregation) {
+    public FieldAccessorPostAggregation(@NotNull MetricField aggregation) {
```

Comment:
Shall we change parameter name to `metricField`?

Reply:
@QubitPi its actually an `aggregation` that will be used here- hence the name